azure/main.tf

```diff
@@ -347,9 +347,12 @@ module "readapi_queue" {
   providers = {
     azurerm = azurerm.readapi
   }
-  source       = "app.terraform.io/indico/indico-azure-readapi-queue/mod"
-  version      = "1.0.0"
-  readapi_name = lower("${var.account}-${var.label}-s")
+  source               = "app.terraform.io/indico/indico-azure-readapi-queue/mod"
+  version              = "1.1.0"
+  readapi_type         = var.readapi_type
+  readapi_name         = var.readapi_type == "create" ? lower("${var.account}-${var.label}-s") : var.readapi_name
```

Comment:
Should also be an override variable, and we should generate the default name inside the root of the module